docs/layers.md

```diff
@@ -2341,6 +2342,7 @@ However, if the roads layer does not include `linear_ref_id`, then it should be
 * `is_tunnel`: set to `true` when the linear is a tunnel
 * `is_link`: set to true when the linear is a slip-road
 * `drives_on_left`: set to `true` when the country drives on the left, e.g. In the U.K
+* `is_express_lane`: set to `true` when the linear is an HOV (High-Occupancy Vechicle) lane.
```

Comment:
See above, would be `is_hov`.

But please explain why HOV tag is needed here at all. Plus we're talking about points here, right? So the comment would also need to update.

Reply:
Please see my comment above :)